hack/dind-cluster.sh

```diff
@@ -67,14 +67,15 @@ function start() {
   local deployed_config_root=$4
   local cluster_id=$5
   local network_plugin=$6
-  local wait_for_cluster=$7
-  local node_count=$8
-  local additional_args=$9
+  local container_runtime=$7
+  local wait_for_cluster=$8
+  local node_count=$9
+  local additional_args=${10}
```

Comment:
This is getting kind of insane... Would it be better to just refer to the global variables (that already exist)?

Reply:
TODO: replace this with a python script one day